plugins/csv/src/main/java/org/itsallcode/whiterabbit/plugin/csv/CSVProjectReportExporter.java

```diff
@@ -0,0 +1,100 @@
+
+import org.itsallcode.whiterabbit.api.features.ProgressMonitor;
+import org.itsallcode.whiterabbit.api.features.ProjectReportExporter;
+import org.itsallcode.whiterabbit.api.model.DayType;
+import org.itsallcode.whiterabbit.api.model.ProjectReport;
+import org.itsallcode.whiterabbit.api.model.ProjectReportActivity;
+import org.itsallcode.whiterabbit.api.model.ProjectReportDay;
+
+class CSVProjectReportExporter implements ProjectReportExporter
+{
+    private final boolean filterForWeekDays;
+    private final OutStreamProvider outStreamProvider;
+    private final String separator;
+
+    CSVProjectReportExporter(boolean filterForWeekDays, String separator, OutStreamProvider outStreamProvider) {
+        this.filterForWeekDays = filterForWeekDays;
+        this.outStreamProvider = outStreamProvider;
+        this.separator = separator;
+    }
+
+    @Override
+    public void export(ProjectReport report, ProgressMonitor progressMonitor)
+    {
+        progressMonitor.setTaskName("Exporting...");
+        if (progressMonitor.isCanceled())
+        {
+            return;
+        }
+        List<ProjectReportDay> filteredDays = filterForWeekDays ?
+                report.getDays().stream().filter(day -> day.getType() == DayType.WORK).collect(Collectors.toList()) :
+                report.getDays();
+
+        progressMonitor.beginTask("Initializing...", report.getDays().size());
+        try {
+            PrintStream os = new PrintStream(outStreamProvider.getStream(report.getMonth().toString()));
+
+            os.println(MessageFormat.format("Date{0}Project{0}Time{0}Comment", separator));
+
+            for (final ProjectReportDay day : filteredDays)
+            {
+                if (progressMonitor.isCanceled())
+                {
+                    return;
+                }
+                progressMonitor.worked(1);
+                final String dayComment = formatEmptyString(day.getComment());
+                os.println(MessageFormat.format("{0}{1}{1}{1}{2}", day.getDate().toString(), separator, dayComment));
+
+                for (final ProjectReportActivity project : day.getProjects())
+                {
+                    progressMonitor.setTaskName(getTaskName(day, project));
+                    if (progressMonitor.isCanceled())
+                    {
+                        return;
+                    }
+                    final String duration = formatDuration(project.getWorkingTime());
+                    final String projectLabel = formatEmptyString(project.getProject().getLabel());
+                    final String activityComment = formatEmptyString(project.getComment());
+                    os.println(MessageFormat.format("{0}{1}{0}{2}{0}{3}",
+                            separator, projectLabel, duration, activityComment));
+                }
+            }
+            os.close();
+
+        } catch (IOException ex) {
+            throw new RuntimeException("Error exporting report to CSV:" + ex.toString());
```

Comment:
Please always pass the cause exception when re-throwing.
In this case, you could also use an UncheckedIOException.

```suggestion
            throw new UncheckedIOException("Error exporting report to CSV:" + ex.toString(), ex);
```